cmd/mirror-url.go

```diff
@@ -102,12 +112,25 @@ func deltaSourceTarget(sourceURL, targetURL string, isFake, isOverwrite, isRemov
 	}
 
 	// List both source and target, compare and return values through channel.
-	for diffMsg := range objectDifference(sourceClnt, targetClnt, sourceURL, targetURL, excludeOptions) {
+	for diffMsg := range objectDifference(sourceClnt, targetClnt, sourceURL, targetURL) {
 		if diffMsg.Error != nil {
 			errorIf(diffMsg.Error, "Unable to mirror objects.")
 			// Ignore error and proceed to next object.
 			continue
 		}
+
+		srcSuffix := strings.TrimPrefix(diffMsg.FirstURL, sourceURL)
+		//Skip the source object if it matches the Exclude options provided
+		if matchExcludeOptions(excludeOptions, srcSuffix) {
+			continue
+		}
+
+		tgtSuffix := strings.TrimPrefix(diffMsg.SecondURL, targetURL)
```

Comment:
It is easy to understand: 'please exclude some files in source when mirroring'.. but it is not easy to understand 'please exclude some files in target when mirroring`.. Is there any useful use case for the latter scenario ?

Reply:
@vadmeste , if --remove flags is passed, the files in the target will be removed. So, exclude options should be applied to both source and target.

Reply:
> @vadmeste , if --remove flags is passed, the files in the target will be removed. So, exclude options should be applied to both source and target.

okay, then you need to update --exclude help message.

Reply:
done